pkg/scheduler/api/compatibility/compatibility_test.go

```diff
@@ -1194,6 +1194,133 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 				}},
 			},
 		},
+		"1.17": {
```

Comment:
Just heads up that you will need to update this after https://github.com/kubernetes/kubernetes/pull/82735 is merged.

Reply:
done